test/common/bond_test.js

```diff
@@ -48,8 +48,20 @@ test('determineBondProvider should return the default BondProvider if the URL ho
     t.is(determineBondProvider('drs://some-host/anything'), BondProviders.default);
 });
 
-test('determineBondProvider should not return a provider if the URL host is JDR"', (t) => {
-    t.falsy(determineBondProvider('drs://jade.datarepo-dev.broadinstitute.org/identifier'));
+test('determineBondProvider should return the "jade-data-repo" as the provider for JDR dev host"', (t) => {
+    t.is(determineBondProvider('drs://jade.datarepo-dev.broadinstitute.org/identifier'), BondProviders.JADE_DATA_REPO);
```

Comment:
There is no actual `BondProviders.JADE_DATA_REPO` variable, so this line evaluates to:

```javascript
t.is(undefined, undefined)
```

IMHO it reads better when evaluated to

```javascript
t.falsy(undefined)
```